src/stores/components/PendingTransactionsSnackbarStore.js

```diff
@@ -0,0 +1,93 @@
+import { observable, action, reaction, runInAction } from 'mobx';
+import _ from 'lodash';
+import { TransactionType, TransactionStatus } from 'constants';
+
+import Transaction from '../models/Transaction';
+import { queryAllTransactions } from '../../network/graphQuery';
+
+
+const INIT_VALUES = {
+  isVisible: false,
+  count: 0,
+  pendingCreateEvents: 0,
+  pendingBets: 0,
+  pendingSetResults: 0,
+  pendingVotes: 0,
+  pendingFinalizeResults: 0,
+  pendingWithdraws: 0,
+  pendingTransfers: 0,
+  pendingResetApproves: 0,
+};
+
+export default class PendingTransactionsSnackbarStore {
+  @observable isVisible = INIT_VALUES.isVisible
+  @observable count = INIT_VALUES.count
+
+  pendingCreateEvents = INIT_VALUES.pendingCreateEvents
+  pendingBets = INIT_VALUES.pendingBets
+  pendingSetResults = INIT_VALUES.pendingSetResults
+  pendingVotes = INIT_VALUES.pendingVotes
+  pendingFinalizeResults = INIT_VALUES.pendingFinalizeResults
+  pendingWithdraws = INIT_VALUES.pendingWithdraws
+  pendingTransfers = INIT_VALUES.pendingTransfers
+  pendingResetApproves = INIT_VALUES.pendingResetApproves
+
+  constructor() {
+    reaction(
+      () => this.count,
+      () => this.isVisible = this.count > 0
+    );
+  }
+
+  @action
+  init = async () => {
+    this.reset();
```

Comment:
Let's do `Object.assign(this, INIT) // reset to initial properties` instead of having a `this.reset()` method

Reply:
done